Serialize/ASN1.cs

```diff
@@ -14,6 +14,13 @@ public class ASN1
         public const uint BACNET_MIN_PRIORITY = 1;
         public const uint BACNET_MAX_PRIORITY = 16;
 
+        /// <summary>
+        /// You can provide a function to resolve a given tag within a property to a different tag (e.g. for custom properties)
+        /// The address of the remote device is provided so that you can support devices from multiple vendors
+        /// (the same custom property# has different meaning for each vendor)
+        /// </summary>
+        public static Func<BacnetAddress, BacnetPropertyIds, byte, BacnetApplicationTags> CustomTagResolver;
```

Comment:
in the future things like this will be injected (so no statics) but for now it's ok (methods are static anyway)

Reply:
yeah... about injection: I really hope you mean a constructor parameter, and not some nifty DI Framework?

Reply:
I don't want to add any dependency to DI frameworks, but the code has to be DI friendly. This means stuff has to be set using constructor parameters, properties, interfaces etc.